src/ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
```

Comment:
Can you send me a few sentences that explain what this document is used for? I can refine the language for you.